db/structure.sql

```diff
@@ -208,7 +208,8 @@ CREATE TABLE public.certificate_template_entries (
     font_size integer DEFAULT 16,
     align integer DEFAULT 0,
     created_at timestamp without time zone NOT NULL,
-    updated_at timestamp without time zone NOT NULL
+    updated_at timestamp without time zone NOT NULL,
+    color character varying
```

Comment:
The structure dump does not represent all the migration changes, the `default` and `null` are absent. Please dump it again.

Reply:
Corrected.